src/list.tsx

```diff
@@ -122,17 +122,19 @@ interface CommonProps {
     extra?: React.ReactNode;
     dataAttributes?: DataAttributes;
     disabled?: boolean;
+    navigable?: boolean; // enables chevron on custom elements with text
 }
 
 interface ContentProps extends CommonProps {
     isClickable?: boolean;
     type?: 'chevron' | 'basic' | 'custom' | 'control';
-    right?: React.ReactNode;
+    right?: ({centerY}: {centerY: boolean}) => React.ReactNode | React.ReactNode;
```

Comment:
I miss some parenthesis here, otherwise this wont work
```suggestion
    right?: (({centerY}: {centerY: boolean}) => React.ReactNode) | React.ReactNode;
```